src/Microsoft.ML.FastTree/FastTreeTweedie.cs

```diff
@@ -78,9 +76,9 @@ public sealed partial class FastTreeTweedieTrainer
         }
 
         /// <summary>
-        /// Initializes a new instance of <see cref="FastTreeTweedieTrainer"/> by using the legacy <see cref="Arguments"/> class.
+        /// Initializes a new instance of <see cref="FastTreeTweedieTrainer"/> by using the legacy <see cref="Options"/> class.
```

Comment:
>legacy [](start = 92, length = 6)

Change legacy.

Reply:
fixed

---
In reply to: [245968568](https://github.com/dotnet/machinelearning/pull/2047#discussion_r245968568) [](ancestors = 245968568)